docs/en/00_Getting_Started/00_Server_Requirements.md

```diff
@@ -35,9 +33,9 @@ Our web-based [PHP installer](installation/) can check if you meet the requireme
 
  * See [phpinfo()](http://php.net/manual/en/function.phpinfo.php) for more information about your environment
  * One of the following databases: 
-   * MySQL 5.0+
-   * PostgreSQL 8.3+ (requires ["postgresql" module](http://silverstripe.org/postgresql-module))
-   * [SQL Server 2008+](http://silverstripe.org/microsoft-sql-server-database/), [Oracle](https://github.com/smindel/silverstripe-oracle) and [SQLite](http://silverstripe.org/sqlite-database/) are not commercially supported, but are under development by our open source community.
+   * MySQL 5.5+
```

Comment:
Unfortunately I couldn't find any official documentation on its end of life. Only some rumors that it may end in December 2018. However, it's still available for download on the website and there are placeholders in the changelog for at least 2 other future releases...

On the other hand, it's only broken in 4.x branch, whereas 4.0, 4.1, 4.2 and 4.3 still pass all tests on MySQL 5.5. Also, I couldn't find any functionality that may be broken, only that single test. So I think if we just ditch the test, we still can keep support for MySQL 5.5 without much trouble.

Reply:
Since we don't release 4.4 yet, we can keep an eye on https://www.mysql.com/support/eol-notice.html and if they drop it, we could drop it too?

Reply:
EOLs seem to be handled by Oracle now, see the references in https://en.wikipedia.org/wiki/MySQL#Release_history (link posted on main ticket about this previously)

Reply:
Yes, but Wikipedia is not an official documentation. I've rather been looking in the MySQL documentation and couldn't find it there. Their [EOL notice page](https://www.mysql.com/support/eol-notice.html) also seems to be up to date. It includes a note about 5.1, but nothing for 5.5 yet. That's what I suggest we keep an eye on. Otherwise 5.5 seems to be still officially supported by Oracle.

Reply:
The reference on Wikipedia points to the [Oracle Lifetime Support Policy](http://www.oracle.com/us/support/library/lifetime-support-technology-069183.pdf), which presumably *is* an official reference. And that Extended Support runs out in 12/2018. We need to distinguish between commercial support through Oracle, and generic support by the Open Source project. The note you reference in the MySQL EOL docs ("MySQL 5.1 covered under Oracle Lifetime Sustaining Support") references Oracle's support model. I'm not sure if the open source project keeps separate EOL deadlines. In the wake of that ambiguity, I don't think the burden is on us to assume they're different from Oracle's.

Reply:
Yes, sorry, I didn't reed the Oracle Lifetime Support Policy, that's why I didn't find the 12/2018 date. I completely agree that the paid support should be the same as EOL to us. I'll update the PR to claim `MySQL 5.6+` then.